src/crimson/osd/osd_operations/snaptrim_event.cc

```diff
@@ -414,65 +414,67 @@ SnapTrimObjSubEvent::remove_or_update(
 SnapTrimObjSubEvent::snap_trim_obj_subevent_ret_t
 SnapTrimObjSubEvent::start()
 {
-  return enter_stage<interruptor>(
+  auto exit_handle = seastar::defer([this] {
```

Comment:
nit:
* [[maybe_unused]]
* `// Called on exit_handle destruction`

Reply:
I don't think it actually needs the [[maybe_unused]] -- the constructor/destructor have side effects.  "Called on exit_handle's destruction" merely explains what seastar::defer does -- that will mostly be unnecessary visual noise for anyone familiar with the code base and fairly easy to pick up from context for anyone with a basic understanding of C++ RAII.